bin/whisper-resize.py

```diff
@@ -172,7 +172,7 @@
 try:
   print('Renaming new database to: %s' % path)
   os.rename(tmpfile, path)
-except (OSError, FileNotFoundError, PermissionError):
+except (OSError):
```

Comment:
This change is because `FileNotFoundError` & `PermissionError` are both subclasses of `OSError`, and are not defined in Python <3.3